src/core/nick-rec.h

```diff
@@ -8,6 +8,7 @@ time_t last_check; /* last time gone was checked */
 char *nick;
 char *host;
 char *realname;
+char *account;
```

Comment:
At what point are we ABI bumping? It'd be annoying and conflicting to do it in the middle of all these PRs, but.

Reply:
yes...